components/header/header.module.scss

```diff
@@ -53,22 +53,22 @@
   background: $color-dark;
   display: flex;
   justify-content: center;
-  opacity: 0.9;
+  opacity: 0;
```

Comment:
hey i suppose opacity: 0 should only be in mobile mode because now the links are not seen in the desktop mode.

Reply:
Got it. Thanks for flagging.